### ISSUE TITLE
Zero loss after feeding the first batch

### ISSUE THREAD

**Issue by mortezamg63** [UM-ARM-Lab/Chamfer-Distance-API#1]
Hello,
I uses other loss function in tensorflow and it works fine. But, when I define Chamfer distance loss function, I get zero loss in all batches except the first one. 
Could you guide me on this problem?

Thanks
